docs/source/dev-guide/plugin-api/subcommand_guide.rst

```diff
@@ -0,0 +1,174 @@
+=================================
+Custom subcommand plugin tutorial
+=================================
+
+In this tutorial, we will create a new ``conda`` subcommand that can convert a string
+into ASCII art.
+
+.. note::
+
+   This tutorial requires the ``art`` package, available on `PyPI`_.
```

Comment:
I am not sure this makes sense as the requirement for this tutorial. Really, what you need is a conda and a conda environment with python + pip installed. Later in the tutorial you explain how to get the `art` package via a `pip` install.

Also, this is just my opinion, but I would not use the "note"/"callout" formatting for this. Those are typically reserved for things that readers are allowed to skip while reading an article. Think about those text books with extra information in these types of boxes that aren't necessarily crucial for grasping concepts.